docker_files_test.go

```diff
@@ -161,9 +150,12 @@ func TestCopyDirectoryToRunningContainerAsFile(t *testing.T) {
 		},
 		Started: true,
 	})
+	require.NoError(t, err)
```

Comment:
The examples I saw on the testcontainer website didn't leak the assertion library, which is why I kind of ignored the errors. Maybe:

```suggestion
if err != nil {
		t.Fatal(err)
	}
```

Would be best here?

Reply:
Mmm you are right, let me revert that single commit 🙏

Reply:
One question: is it fine to leak the testing library?